src/modules.ts

```diff
@@ -0,0 +1,49 @@
+/*
+  Copyright 2019 Esri
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import { getScript, ILoadScriptOptions, isLoaded, loadScript } from './script';
+import utils from './utils/index';
```

Comment:
it might just be superstition (or lack of understanding), but for some reason i feel like its better to _always_ destructure import statements.

```ts
import { utils } from './utils/index';
```